tileserver/__init__.py

```diff
@@ -234,7 +253,8 @@ def handle_request(self, request):
         if (self.health_checker and
                 self.health_checker.is_health_check(request)):
             return self.health_checker(request)
-        request_data = parse_request_path(request.path, self.extensions)
+        request_data = parse_request_path(request.path, self.extensions,
+                                          styles=self.styles)
```

Comment:
You can just pass `self.styles` in here if you want. Setting the keyword there is maybe redundant.